Puppetfile.lock

```diff
@@ -118,42 +105,13 @@ FORGE
     thias-squid3 (1.0.2)
       puppetlabs-stdlib (>= 2.4.0)
 
-GIT
-  remote: https://github.com/Katello/puppet-katello_devel
-  ref: master
-  sha: ac9228481d8d76b87e655e76f5fae317330dfaf4
-  specs:
-    katello-katello_devel (1.0.0)
-      katello-candlepin (< 5.0.0, >= 4.0.0)
-      katello-certs (< 4.0.0, >= 2.0.0)
-      katello-katello (< 8.0.0, >= 6.1.0)
-      katello-pulp (< 6.0.0, >= 4.3.0)
-      katello-qpid (< 4.0.0, >= 3.0.0)
-      puppet-extlib (< 3.0.0, >= 0.10.4)
-      puppetlabs-apache (< 3.0.0, >= 1.0.0)
-      puppetlabs-postgresql (< 6.0.0, >= 3.0.0)
-      puppetlabs-stdlib (< 5.0.0, >= 4.0.0)
-      puppetlabs-vcsrepo (< 3.0.0, >= 1.0.0)
-
 DEPENDENCIES
-  katello-candlepin (< 4.1.0, >= 4.0.0)
+  katello-candlepin (< 5.1.0, >= 5.0.0)
   katello-certs (< 3.1.0, >= 3.0.0)
   katello-common (< 1.1.0, >= 1.0.1)
-  katello-foreman_proxy_content (< 6.1.0, >= 6.0.0)
-  katello-katello (< 7.1.0, >= 7.0.0)
-  katello-katello_devel (>= 0)
-  katello-pulp (< 5.1.0, >= 5.0.0)
-  katello-qpid (< 3.2.0, >= 3.1.0)
+  katello-foreman_proxy_content (< 6.2.0, >= 6.1.0)
+  katello-katello (< 7.3.0, >= 7.2.0)
+  katello-pulp (< 5.4.0, >= 5.3.0)
+  katello-qpid (< 4.1.0, >= 4.0.0)
   katello-service_wait (< 1.1.0, >= 1.0.3)
-  puppetlabs-mysql (< 4.1.0, >= 4.0.0)
```

Comment:
Could you run the update with these modules in the Puppetfile so they stay locked on the right versions?